exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -390,4 +391,15 @@
 
   String DYNAMIC_UDF_SUPPORT_ENABLED = "exec.udf.enable_dynamic_support";
   BooleanValidator DYNAMIC_UDF_SUPPORT_ENABLED_VALIDATOR = new BooleanValidator(DYNAMIC_UDF_SUPPORT_ENABLED, true, true);
+
+  /**
+   * Option whose value is a long value representing the number of bits required for computing ndv (using HLL)
+   */
+  LongValidator NDV_MEMORY_LIMIT = new PositiveLongValidator("exec.statistics.ndv_memory_limit", 30, 20);
+
+  /**
+   * Option whose value represents the current version of the statistics. Decreasing the value will generate
+   * the older version of statistics
+   */
+  LongValidator STATISTICS_VERSION = new NonNegativeLongValidator("exec.statistics.capability_version", 1, 1);
```

Comment:
Say in the next version(v2), we add histograms. Computing stats is expensive so users might prefer to remain on the present version(v1) maybe because their queries do not involve too many inequalities. Always generating the latest version of the stats will force the users to compute the latest and greatest stats without needing them. On the other hand, providing individual control of which statistic to compute moves too much burden onto the user to figure out exactly which statistics would help their use-cases.